stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -123,20 +134,211 @@ The `description` property allows you to provide additional information. It can
+
+If a listItem does not have a `onClick` property it will not have this behaviour.
+
+<Canvas>
+  <Story name="On click">
+    {() => {
+      const [isOpened, setIsOpened] = useState(false)
+      const handleClick = () => setIsOpened(current => !current)
+      return (
+        <>
+          <List layout="grid">
+            <ListItem
+              title={
+                <Typography color="inverted-text" fontWeight="bold">
+                  ØKP4
+                </Typography>
+              }
+              description={
+                <Typography
+                  as="div"
+                  style={{
+                    display: 'grid',
+                    textAlign: 'center',
+                    alignItems: 'center',
+                    height: '100%'
+                  }}
+                  color="inverted-text"
+                >
+                  The Next Generation of Data Applications
+                </Typography>
+              }
+              lastElement={
+                <Typography
+                  as="div"
+                  color="inverted-text"
+                  fontSize="x-small"
+                  fontWeight="x-light"
+                  style={{ textAlign: 'right' }}
+                >
+                  Clickable
+                </Typography>
+              }
+              onClick={handleClick}
+            />
+            <ListItem
+              title={
+                <Typography color="inverted-text" fontWeight="bold">
+                  ØKP4
+                </Typography>
+              }
+              description={
+                <Typography
+                  as="div"
+                  style={{
+                    display: 'grid',
+                    textAlign: 'center',
+                    alignItems: 'center',
+                    height: '100%'
+                  }}
+                  color="inverted-text"
+                >
+                  The Next Generation of Data Applications
+                </Typography>
+              }
+              lastElement={
+                <Typography
+                  as="div"
+                  color="inverted-text"
+                  fontSize="x-small"
+                  fontWeight="x-light"
+                  style={{ textAlign: 'right' }}
+                >
+                  Not clickable
+                </Typography>
+              }
+            />
+          </List>
+          <Toast
+            isOpened={isOpened}
+            onOpenChange={handleClick}
+            title="ListItem clicked !"
+            severityLevel="success"
+          />
+        </>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## First and last elements
 
-The `leftElement` and `rightElement` properties allow you to provide any `JSX.Element` as icons, buttons, text, etc. to an item in the list.  
+The `firstElement` and `lastElement` properties allow you to provide any `JSX.Element` as icons, buttons, text, etc. to an item in the list.  
```

Comment:
It bothers me that you never speak about the position of each element, so, except their name, what's the real difference between those 2 properties?

Reply:
I'll add some description about their position 😺